frontend/public/components/monitoring/alerting.tsx

```diff
@@ -1380,6 +1380,18 @@ const MonitoringListPage: React.FC<ListPageProps & {
             {silencesLoadError.json?.error || silencesLoadError.message}
           </PFAlert>
         )}
+        {loadError && (
```

Comment:
`loadError` can be the load error for the alerts, the rules or the silences depending on the page.

We already have `silencesLoadError` available, so I think we just need to use that. In other words, I think we just need to change the `silencesLoadError && !loadError` case above.

Reply:
Good catch @kyoto . I realized the `loadError` in some cases is the same value as `silencesLoadError`, in particular when you are in the silences tab. I refactored the code to lift the Alert component so the consumers of the `MonitoringListPage` component are able to decide which alert to display. WDYT? 👀

Reply:
💯